### PR TITLE
feat: Add new module standard webtests

### PR DESCRIPTION
**Pull request by re-sh-cloud-arch**
### Type of changes

- [ ] Add new module
- [ ] Update existing module
- [ ] Remove existing module

### Does this introduce a breaking change?

- [ ] Yes
- [ ] No

### Run checks

Useful commands to run checks on local machine

```sh
bash .utils/terraform_run_all.sh init local
pre-commit run -a
```

### CODE REVIEW

File: application_insights_standard_web_test/main.tf [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+
+## migrate to Microsoft.Insights/webTests 
+
+resource "azurerm_monitor_metric_alert" "alert_this" {
+  name                = local.alert_name
+  resource_group_name = var.application_insights_resource_group
+  scopes              = [var.application_insights_id]
+  description         = "Whenever the average availabilityresults/availabilitypercentage is less than ${var.https_probe_threshold}%"
+  severity            = 0
+  frequency           = "PT5M"
```
Comment (diegolagospagopa) [inline application_insights_standard_web_test/main.tf:46]: can be a variable to override?

File: application_insights_standard_web_test/main.tf [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+  resource_group_name     = var.application_insights_resource_group
+  location                = var.location
+  application_insights_id = var.application_insights_id
+  geo_locations           = ["emea-nl-ams-azr"]
+  description             = "HTTP Standard WebTests"
+  frequency               = var.frequency
+  enabled                 = var.alert_enabled
+
+  request {
+    url       = format("%s%s", var.https_endpoint, var.https_endpoint_path)
```
Comment (diegolagospagopa) [inline application_insights_standard_web_test/main.tf:22]: is possible to use string interpolation?

**Review by diegolagospagopa — CHANGES_REQUESTED** [review]

File: application_insights_standard_web_test/main.tf (hunk 1 above)
Comment (re-sh-cloud-arch) [reply application_insights_standard_web_test/main.tf]: check now

File: application_insights_standard_web_test/main.tf (hunk 2 above)
Comment (re-sh-cloud-arch) [reply application_insights_standard_web_test/main.tf]: check now

**Review by diegolagospagopa — APPROVED** [review]

**Pull request merged**